test/fuzz/transformation_replace_id_with_synonym_test.cpp

```diff
@@ -1888,7 +1888,7 @@ TEST(TransformationReplaceIdWithSynonymTest,
 //  test so that it covers more atomic operations, and enable the test once the
 //  issue is fixed.
 TEST(TransformationReplaceIdWithSynonymTest,
-     DISABLED_SignOfAtomicScopeAndMemorySemanticsDoesNotMatter) {
+     SignOfAtomicScopeAndMemorySemanticsDoesNotMatter) {
   // TODO(https://github.com/KhronosGroup/SPIRV-Tools/issues/4345): both the
   //  GLSL comment and the corresponding SPIR-V should be updated to cover a
   //  larger number of atomic operations.
```

Comment:
I think we need to keep this test disabled at the moment.

The reason is that #4349 is going to make it illegal to replace a memory semantics or memory scope operand, so replacing them with constants of a different sign won't be allowed anymore by spirv-fuzz (even though it is legal).

We can consider returning to this particular test in the future.

Reply:
All right!